packages/common/src/lib/common/tools.ts

```diff
@@ -466,7 +470,7 @@ function updateUuid(newUuid: string, _objects: any, callback: (uuid?: string) =>
         _objects.setObject(
             'system.meta.uuid',
             {
-                type: 'meta',
```

Comment:
For me personally it is a bit weird, because of the id itself containing meta^^

Reply:
It is historical